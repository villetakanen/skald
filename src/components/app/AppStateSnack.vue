<template>
  <v-snackbar></v-snackbar>
</template>
