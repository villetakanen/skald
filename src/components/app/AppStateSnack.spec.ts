import { shallowMount } from '@vue/test-utils'
import AppStateSnack from './AppStateSnack.vue'

describe('Mounted App', () => {
  const wrapper = shallowMount(AppStateSnack)

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
