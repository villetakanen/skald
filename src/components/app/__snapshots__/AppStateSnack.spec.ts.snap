// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted App renders correctly 1`] = `
<v-snackbar-stub
  contentclass=""
  tag="div"
  timeout="5000"
  transition="v-snack-transition"
/>
`;
